src/content/en/agencies/partials/web-directory.html

```diff
@@ -0,0 +1,31 @@
+{% framebox class="directory-framebox" height="auto" enable_widgets="true" %}
```

Comment:
this file should be in `src/content/en/agencies/_partials/` - note the underscore on partials. That prevents files in this directory from being published by themselves. They can only be published when they're included in something else.

Reply:
Fixed